sql/synth_recipes.sql

```diff
@@ -3195,6 +3195,20 @@ INSERT INTO `synth_recipes` VALUES (49946,1,0,0,0,0,0,24,0,0,0,4100,4242,12442,0
 INSERT INTO `synth_recipes` VALUES (49947,1,0,0,0,0,0,29,0,0,0,4100,4242,12993,0,0,0,0,0,0,0,850,850,848,848,1,1,1,1,'Sandals (desynth)');  -- 2021.11.15 Based on synth.  Sites that show are in agreement.
 INSERT INTO `synth_recipes` VALUES (49948,1,0,0,0,0,0,85,0,0,0,4100,4242,13595,0,0,0,0,0,0,0,820,863,863,863,1,1,1,1,'Coeurl Mantle (desynth)');
 INSERT INTO `synth_recipes` VALUES (49949,1,0,0,0,0,0,34,0,0,0,4098,4240,1638,0,0,0,0,0,0,0,850,850,850,850,1,1,1,1,'Moblin Mask (desynth)');  -- 2021.11.15 JPWiki and FFXIclopedia show level 34
+INSERT INTO `synth_recipes` VALUES (49950,1,0,0,0,0,0,26,0,0,0,4100,4242,12698,0,0,0,0,0,0,0,848,651,651,651,1,1,2,2,'Studded Gloves (desynth)');  -- 2022.03.01 Base on FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (49951,1,0,0,0,0,0,32,0,0,0,4100,4242,12570,0,0,0,0,0,0,0,848,651,848,851,1,2,1,1,'Studded Vest (desynth)'); -- Based on normal desynth behavior
+INSERT INTO `synth_recipes` VALUES (49952,1,0,0,4,0,0,3,0,0,0,4100,4242,12448,0,0,0,0,0,0,0,649,649,850,850,1,1,1,2,'Bronze Cap (desynth)'); -- 2022.03.01 Base on FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (49953,1,0,0,0,0,0,5,22,0,0,4100,4242,12966,0,0,0,0,0,0,0,880,850,850,893,1,1,2,1,'Bone Leggings (desynth)');  -- 2022.03.01 Base on FFXIclopedia
+INSERT INTO `synth_recipes` VALUES (49954,1,0,0,0,0,0,5,20,0,0,4100,4242,12710,0,0,0,0,0,0,0,880,880,850,850,1,2,1,2,'Bone Mittens (desynth)'); -- 2022.03.01 Base on normal desynth behavior
```

Comment:
Bone Mittens
No info on XiClopedia for this recipe. If you found info, please provide a link^^
For now though, comment it out until we can get some more info or retail confirmation.